src/crimson/osd/object_context.h

```diff
@@ -74,6 +75,10 @@ class ObjectContext : public ceph::common::intrusive_lru_base<
   using watch_key_t = std::pair<uint64_t, entity_name_t>;
   std::map<watch_key_t, seastar::shared_ptr<crimson::osd::Watch>> watchers;
 
+  // attr cache. ECTransaction is the initial user
+  using attr_cache_t = std::map<std::string, ceph::buffer::list, std::less<>>;
```

Comment:
This looks like https://github.com/ceph/ceph/pull/66463, should we try merging it so this won't PR won't be blocked?